web_monitoring/diffing_server.py

```diff
@@ -57,10 +58,24 @@
 
 client = tornado.httpclient.AsyncHTTPClient()
 
+acao_env_value = \
+    os.environ.get('ACCESS_CONTROL_ALLOW_ORIGIN_HEADER', '-1')
```

Comment:
Why the string `'-1'` insted of `None` here? That would simplify the code here (`os.environ.get('ACCESS_CONTROL_ALLOW_ORIGIN_HEADER')` — no default needed) and simplify the check in `set_default_headers()`.

Also, please don’t use `acao` in the name here — we try to [only use abbreviations for words that are *primarily* referred to by their abbreviation (e.g. URL, FAQ)](https://github.com/edgi-govdata-archiving/web-monitoring/blob/master/CONTRIBUTING.md#naming). Doing so makes it a lot easier to understand for someone else reading the code later.